### PR TITLE
Wrote new python script to compute averages from a set of images to r…

### PR DESCRIPTION
**Pull request by Gab-D-G**
…eplace the currently slow median implementation.

### CODE REVIEW

File: modelbuild_averager.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                        """)
+    parser.add_argument("--method", default='trimmed_mean',
+                        choices=['mean', 'median', 'trimmed_mean', 'huber'],
+                        help="""
+                        Specify of average method to create from the image list.
+                        """)
+    parser.add_argument("--trim_percent", type=float, default=0.15,
+                        help="""
+                        Specify % to trim off if using trimmed_mean.
+                        """)
```
Comment (gdevenyi) [inline modelbuild_averager.py:33]: Trim fraction? I like you switched to fraction as requested, but the help is now misleading.

File: modelbuild_averager.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    parser.add_argument('--file_list', type=str,
+                        nargs="*",  # 0 or more values expected => creates a list
+                        help="""
+                        Specify a list of input files, space-seperated (i.e. file1 file2 ...).
+                        """)
+    parser.add_argument("--image_type", default='image',
+                        choices=['image', 'warp'],
+                        help="""
+                        Specify whether the type of image is a nifti structural image,
+                        or a set of affine or non-linear (warp) transforms.
```
Comment (gdevenyi) [inline modelbuild_averager.py:23]: > set of affine

this code doesn't do affines.

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by Gab-D-G** [comment]
is this alright?
```python
elif opts.method == 'efficient_trimmed_mean':
    # computes the average from the 20th,50th and 80th percentiles
    average = np.quantile(concat_array, (0.2,0.5,0.8),axis=0).mean(axis=0)
```

**Comment by gdevenyi** [comment]
Yes, except its not a trimmed mean, its a "trimean". Please include the wikipedia link in the  comment.

**Review by gdevenyi — APPROVED** [review]

**Pull request merged**